packages/apollo-server-codegen/src/IR/Enums.ts

```diff
@@ -0,0 +1,18 @@
+import { EnumTypeDefinitionNode } from "graphql";
+import { Translatable, Translator } from "../Translators";
+import { Description } from "./Descriptions";
+
+export class EnumDefinition implements Translatable {
```

Comment:
Should `Translatable` take an ASTNode type generic for the types it supports? In this case `Translatable<EnumTypeDefinitionNode>`

Reply:
We could add a type parameter like that, but what would it actually do?

Reply:
I'm not sure the generic makes sense here, as there a some non-ASTNode things that also implement `Translatable`.